package.json

```diff
@@ -54,9 +54,10 @@
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-3": "^6.24.1",
     "babel-runtime": "^6.26.0",
-    "bclient": "^0.0.1",
+    "bclient": "^0.1.0",
     "bcoin": "git+https://github.com/bcoin-org/bcoin.git",
     "bcurl": "git+https://github.com/bucko13/bcurl.git#update-brq",
+    "bledger": "0.0.2",
```

Comment:
Does this need to be in bpanel? We're not using it anywhere else in this repo are we? It's more of just a dependency for the wallet.